### ISSUE TITLE
Password incorrect

### ISSUE THREAD

**Issue by pravic** [ytisf/theZoo#45]
theZoo\malwares\Binaries\Dropper.Taleret\Dropper.Taleret.zip
theZoo\malwares\Binaries\Trojan.Tapaoux\Trojan.Tapaoux.zip

**Comment by neutt22** [comment]
have you triend infected

**Comment by pravic** [comment]
Yep. "infected" and "infected " (second is in Dropper.Taleret.pass)

**Comment by Gaurav-Mall** [comment]
This fixes #45.

**Comment by PierrickV** [comment]
Dropper.Taleret.zip --> infected - Wrong Password
+1

**Comment by NotAProDeveloper** [comment]
> Dropper.Taleret.zip --> infected - Wrong Password
> +1

and whats the correct password ? :D
